pkg/artifactory/resource/repository/virtual/virtual.go

```diff
@@ -26,7 +26,7 @@ type RepositoryBaseParams struct {
 
 type RepositoryBaseParamsWithRetrievalCachePeriodSecs struct {
 	RepositoryBaseParams
-	VirtualRetrievalCachePeriodSecs int `hcl:"retrieval_cache_period_seconds" json:"virtualRetrievalCachePeriodSecs"`
+	VirtualRetrievalCachePeriodSecs int `hcl:"retrieval_cache_period_seconds" json:"virtualRetrievalCachePeriodSecs,omitempty"`
```

Comment:
Is `omitempty` necessary?

Reply:
Yes, otherwise it will send `0` as an attribute value if it's not explicitly set. The behavior we want is to not send anything, so the value will be defined by RT.